operating/manage-user-access/authentication-provider-structure.adoc

```diff
@@ -0,0 +1,86 @@
+:_content-type: ASSEMBLY
+[id="authentication-provider-structure"]
+= Authentication provider structure
+include::modules/common-attributes.adoc[]
+:context: authentication-provider-structure
+
+toc::[]
+
+[role="_abstract"]
+
+An authentication provider connects to a third-party source of user identity (for example, an identity provider, or IDP). An authentication provider gets the user identity from IDP, issues a token based on this identity, and returns the token to the caller. The mentioned token is then sent within the user interface or API calls and allows {product-title-short} to authorize the user.
+
+Navigate to *Operating -> Managing user access*, and select the appropriate module to set up your own IDP immediately after provisioning.
+
+[id="auth-provider-claim-mappings"]
+== Claim mappings
+Claim mappings allow a customer to specify if an attribute in the user identity obtained from the IDP should be added to the {product-title-short}-issued token and with which attribute name.
+An example of claim mapping is the mapping from `roles` in the user identity to `groups` in the {product-title-short}-issued token.
+There are different default claim mappings for each auth provider type.
```

Comment:
```suggestion
A claim is the data an identity provider includes about a user inside the token they issue. 

Using claim mappings, you can specify if RHACS should customize the claim attribute it receives from an IDP to another attribute in the {product-title-short}-issued token. 

For example, you can map from `roles` in the user identity to `groups` in the {product-title-short}-issued token using claim mapping.

RHACS uses different default claim mappings for every authentication provider.
```

Reply:
The point that would be important to make is that without claim mapping, a claim from IdP will not be added to the ACS-issued token.

Reply:
Thank you @ivan-degtiarenko we can add the following after the `Using claim mapping ...` sentence:
```
If you do not use the claim mapping, {product-title-short} does not include the claim attribute in the {product-title-short}-issued token.
```